plugins/web/ui/src/routes/status/+page.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+	import { useQuery } from '@sveltestack/svelte-query';
```

Comment:
All of the frontend files use tabs rather than 2 spaces for indentation, which goes against our style guidelines.